wmn-data.json

```diff
@@ -2066,7 +2066,7 @@
         "uri_check" : "https://foursquare.com/{account}",
         "e_code" : 200,
         "e_string" : "on Foursquare</title>",
-        "m_string" : "Foursquare - Independent Location Data Platform",
+        "m_string" : "302 Found</title>",

```

Comment:
I'm not seeing this in the response. Can you confirm and explain how you are checking this?

Reply:
For me, it's redirecting to the home page. With the page title: "Location Technology Unlocking Powerful Connections | FSQ"

Reply:
Tested with node (axios) and python (requests) and received the same. Redirects to home with: <title>Unlocking Powerful Consumer Connections with Location | FSQ</title>